praw/reddit.py

```diff
@@ -151,6 +152,8 @@ def __init__(
         config_interpolation: Optional[str] = None,
         requestor_class: Optional[Type[Requestor]] = None,
         requestor_kwargs: Dict[str, Any] = None,
+        *,
```

Comment:
Should we make `requestor_class` and `requestor_kwargs` keyword only while we're at it?

Reply:
Yes, we can make a separate PR to deprecate them as positional arguments, and I'd add config_interpolation to that list. I don't think it's worth trying to raise a deprecation warning for it, however.